### DISCUSSION TITLE
"Name or service not known" error

### DISCUSSION THREAD

**Discussion by dmitriy-8gears** [inspircd/inspircd#2090] [Support]
So, I'm getting error in different IRC clients, says "Name or service not known". I'm running it by `inspircd --config ~/.config/inspircd/inspircd.conf`. And it runs without single error, but when i try to connect IRC clients says that:

weechat:
│17:27:55 private  -- │ irc: connecting to server irc.private.chat/6667...
│17:27:55 private =!= │ irc: address "irc.private.chat" not found
│17:27:55 private =!= │ irc: error: Name or service not known
│17:27:55 private  -- │ irc: reconnecting to server in 10 seconds

irssi:
Irssi: Looking up irc.private.chat
Irssi: Unable to connect server irc.private.chat port 6667 [Name or service not known]

hexchat:
»» Looking up irc.private.chat
»» Unknown host. Maybe you misspelled it?

my config file looks like this:
https://pastebin.com/raw/SdmeadXC

last version is installed:
`./bin/inspircd --version:`
`InspIRCd-3.17.0-archlinux`

I tried run it both on windows and wsl arch, result the same
VPNs is turned off

**Comment by sadiepowell** [comment]
That sounds like a DNS error. You need to configure the DNS server of your domain to have an A or AAAA record pointing to your server IP address.

**Reply by dmitriy-8gears** [reply]
I'm running it locally on WSL Arch. Is here an option in config file to change?

**Reply by sadiepowell** [reply]
This isn't an IRCd setting. You need to configure it in the DNS settings of your domain provider.

**Reply by dmitriy-8gears** [reply]
Oh, sorry, it was a stupid question.
